src/core/analytics.js

```diff
@@ -293,6 +297,13 @@ class Analytics {
         suffix = '-staging'; // stagging suffix
       }
 
+      if (this.bufferDataPlaneEventsUntilReady) {
+        // Fallback logic to process buffered cloud mode events if integrations are failed to load in given interval
+        setTimeout(() => {
```

Comment:
this does not feel right. Should we not wait until isInitialized is fired? maybe inside replayEvents method?

Reply:
If *all* integrations are not loaded, we have to deliver the events to the data plane as is.